File: delectable/src/main/java/com/delectable/recipe/RecipeService.java
package com.delectable.recipe;

import com.delectable.shared.crud.CRUSoftDeleteRepository;
import com.delectable.shared.crud.CRUSoftDeleteService;

public class RecipeService extends CRUSoftDeleteService<Recipe> {
    public RecipeService(CRUSoftDeleteRepository<Recipe> repository) {
        super(repository);
    }
}

File: delectable-client/src/app/delectable/recipe/service/recipe.service.ts
import { Injectable } from '@angular/core';
import { HttpClient } from '@angular/common/http';
import { CRUDService } from 'app/delectable/shared/service/crud.service';

const recipeAPIEndpoint: string = "/api/recipe";

@Injectable({
  providedIn: 'root'
})
export class RecipeService extends CRUDService {

  constructor(http: HttpClient) {
    super(http, recipeAPIEndpoint)
  }

}
